packages/server/src/csp.ts

```diff
@@ -22,11 +24,15 @@ const vimeoCdn = '*.vimeocdn.com'; // used for video preview images
 const hotjarCom = '*.hotjar.com';
 const hotjarIo = '*.hotjar.io';
 const taskAnalytics = '*.taskanalytics.com';
+const googleFonts = '*.googleapis.com';
```

Comment:
For å få det til å funke med testing av integrasjon lokalt, fjerner når vi får gjort tilpassningene for CDN.